src/qibotn/tebd.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import quimb.tensor as qtn
+from qibo.config import raise_error
+
+
+def init_state_tn_tebd(initial_state):
+    """Creates a inital MPS from a binary string."""
+
+    initial_state = qtn.MPS_computational_state(initial_state)
+    return initial_state
```

Comment:
Is there a strong reason to have a function that just calls a function?

Apart from the documentation, this function definition is 100% equivalent to:
```suggestion
init_state_tn_tebd = qtn.MPS_computational_state
```
i.e. an alias.